scripts/generate_asm.py

```diff
@@ -0,0 +1,73 @@
+import sys
```

Comment:
Can we add a make target or bash script that receives a file name and uses this script to generate a test file? 

Some ideas here: https://stackoverflow.com/questions/2214575/passing-arguments-to-make-run